.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.0
```

Comment:
If we are moving to `2.6.0`, do we need to update our apps to use this version as well? In looking I can see at least `site` is on `2.5.3`.

Reply:
Nope, that shouldn't be necessary. 2.6.0 in influencing the version of ruby that's running Rubocop locally, but not the version of Ruby running in dependencies.